src/client.js

```diff
@@ -36,6 +36,7 @@ module.exports = class Client {
     this._method = 'GET'
     this._path = '/'
 
+    this._disableCase = false
```

Comment:
Instead of doing a negative bool, I'd do this instead:
```suggestion
    this._bodyCasing = true
```